main/Makefile

```diff
@@ -0,0 +1,17 @@
+# Usage:
+# make          # compile all
+# make clean    # remote ALL binaries and objects
+
+# This is an example comment.
```

Comment:
I think this can be derived from all of the other comments.

Reply:
Resolved